src/main/scala/com/gu/automation/support/LocalStorageManager.scala

```diff
@@ -0,0 +1,25 @@
+package com.gu.automation.support
+
+import org.openqa.selenium.{JavascriptExecutor, WebDriver}
+
+import scala.util.Try
+
+object LocalStorageManager {
+
+  private def stripQuotes(input: String): String = {
+    return input filterNot ("'\"" contains _)
```

Comment:
actually if you're going to go to this trouble, you'd be better to escape them as stripping bad characters is insecure in general (however I know this is close test code so I wasn't complaining too much about the original!) ;)